data/docmaps/regression_test/docmap_by_manuscript_id/80984.json

```diff
@@ -253,6 +253,18 @@
                             ]
                         }
                     ]
+                }, {
+                    "participants": [],
+                    "outputs": [{
+                        "type": "computer-file",
```

Comment:
As per the [docmap framework documentation](https://docmaps.knowledgefutures.org/pub/sgkf1pqa#convention-summary), the output should be an [`expression`](https://sparontologies.github.io/fabio/current/fabio.html#d4e3239), but [`computer-file`](https://sparontologies.github.io/fabio/current/fabio.html#d4e2608) is a manifestation.

So output should recreate the values from the [draft RPP Thing](https://github.com/elifesciences/data-hub-api/pull/162/files#diff-71bfaaf7453740566a15d1f8cc5640f283ec98f62f187dbed739f1b197bcc341R24-R30), and attach `content` to that duplicated item.